java/code/src/com/redhat/rhn/frontend/servlets/PxtSessionDelegateImpl.java

```diff
@@ -204,7 +204,7 @@ private void refreshPxtSession(HttpServletRequest request,
         pxtSession.setExpires(TimeUtils.currentTimeSeconds() +
                 SessionManager.lifetimeValue());
         savePxtSession(pxtSession);
-
+        request.getSession().setAttribute("webUserID", pxtSession.getWebUserId());
```

Comment:
Thought experiment: are there cases in which `getSession` could possibly return `null` here?

Reply:
according to the java documentation, the method getSession without any parameter always returns a session: "Returns the current session associated with this request, or if the request does not have a session, creates one." (https://tomcat.apache.org/tomcat-5.5-doc/servletapi/javax/servlet/http/HttpServletRequest.html#getSession())